plugins/plugin-java/che-plugin-java-ext-debugger-java-client/pom.xml

```diff
@@ -130,6 +130,9 @@
         </dependency>
     </dependencies>
     <build>
+        <sourceDirectory>src/main/java</sourceDirectory>
```

Comment:
Why do you add this? Maybe we can remove it because he already have /src/main/java here https://github.com/eclipse/che/pull/807/files#diff-f39aa944deb3f55e7e7d83fb1c51ab93R138